backend/python/app/graphql/queries/language_query.py

```diff
@@ -1,9 +1,12 @@
+from ...middlewares.auth import require_authorization_by_role_gql
 from ..service import services
 
 
+@require_authorization_by_role_gql({"User", "Admin"})
 def resolve_languages(root, info):
     return services["language"].get_languages()
 
 
+@require_authorization_by_role_gql({"User", "Admin"})
 def resolve_is_rtl(root, info, language):
```

Comment:
Not sure if this can be admin only.

Reply:
users need to know if story translation is rtl on the translation platform, so user+admin is fine